src/VisualStudio/IntegrationTest/New.IntegrationTests/InProcess/EditorInProcess.cs

```diff
@@ -176,6 +368,17 @@ public async Task InvokeCodeActionListAsync(CancellationToken cancellationToken)
             await TestServices.Workspace.WaitForAsyncOperationsAsync(FeatureAttribute.SolutionCrawler, cancellationToken);
             await TestServices.Workspace.WaitForAsyncOperationsAsync(FeatureAttribute.DiagnosticService, cancellationToken);
 
+            if (Version.Parse("17.1.31916.450") > await TestServices.Shell.GetVersionAsync(cancellationToken))
+            {
+                // Workaround for extremely unstable async lightbulb prior to:
+                // https://devdiv.visualstudio.com/DevDiv/_git/VS-Platform/pullrequest/361759
```

Comment:
can we track removing this somehow?

Reply:
This code will disable itself at the appropriate time when integration test machines update, so I probably wouldn't bother tracking separately. If we file an issue, it will likely just end up in the backlog and never get seen again, where it may or may not be updated after someone eventually removes this block.

I'd probably take a different approach if this was production code.